opentelemetry-otlp/src/exporter/tonic/logs.rs

```diff
@@ -5,9 +5,8 @@ use opentelemetry_proto::tonic::collector::logs::v1::{
 };
 use opentelemetry_sdk::error::{OTelSdkError, OTelSdkResult};
 use opentelemetry_sdk::logs::{LogBatch, LogExporter};
-use std::sync::Arc;
+use std::sync::{Arc, Mutex};
```

Comment:
are there any repercussions of moving to std Mutex instead of tokio one inside the async export call?